Ollamac/Views/Subviews/CodeHighlighter.swift

```diff
@@ -22,10 +22,12 @@ struct CodeHighlighter: CodeSyntaxHighlighter {
     }
     
     func highlightCode(_ code: String, language: String?) -> Text {
-        guard let highlightedCode = highlightr.highlight(code, as: language) else {
+        guard language?.isEmpty == false,
```

Comment:
Nice catch! One suggestion: it would be useful to automatically set the highlight code if the language isn't specified. Here's an example implementation:

```swift
func highlightCode(_ code: String, language: String?) -> Text {
    let highlightedCode: NSAttributedString?

    if let language, !language.isEmpty {
        highlightedCode = highlightr.highlight(code, as: language)
    } else {
        highlightedCode = highlightr.highlight(code)
    }

    guard let highlightedCode else { return Text(code) }

    var attributedCode = AttributedString(highlightedCode)
    attributedCode.font = .system(size: 16, design: .monospaced)

    return Text(attributedCode)
}
```

Reply:
Yes, you are right. Language detection should take over when the language is not set.
Thanks... I fixed it with your version.